### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by alexpasta**
Error handling in Go

### CODE REVIEW

File: error_handling/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+```swift
+// Option 1
+do {
+  // happy flow
+} catch (let error as NSError) {
+  // error handling
+}
+
+// Option 2: assign to an optional variable
+let response = try? ETPhoneHome("Halp!")
```
Comment (sishichen) [inline error_handling/README.md:22]: typo "help"?

Comment (alexpasta) [reply error_handling/README.md:22]: just copied from the article. it's another way to spell help (將錯就錯)

Comment (sishichen) [reply error_handling/README.md:22]: lol ok

**Review by sishichen — APPROVED** [review]

**Comment by sishichen** [comment]
@alexpasta please add the ~~delete line~~ above the articles you present in the README.md

**Pull request merged**